cmd/emp/dynos.go

```diff
@@ -58,6 +58,9 @@ func listDynos(w io.Writer) {
 func listDyno(w io.Writer, d *heroku.Dyno) {
 	listRec(w,
 		d.Name,
+		d.Id,
+		d.ContainerInstanceID,
+		d.EC2InstanceID,
```

Comment:
Just to keep this de-coupled from AWS specifics, let's call this `Instance` or `Host` (think I prefer `Host`). Could also make this a struct, so we can expand on it later:

```go
type Host struct {
        Id string `json:"id"`
}

type Dyno struct {
....
        Host Host `json:"host"`
....
}
```